Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -13,6 +13,7 @@ import Foundation
 internal class XPCServiceServer: XPCServer {
     
 	private static let service = XPCServiceServer()
+    private static var connection: xpc_connection_t? = nil
```

Comment:
That was my first instinct, but I had an issue around the closure to `xpc_main` being `@convention(c)` (so it's not really actually a closure, it can't close over unique state, like `self`), so I couldn't use `self.connection = connection`.

I think this could work if I reference it using `XPCServiceServer.service.connection = connection`. I'll try it out.